### DISCUSSION TITLE
Regarding bookmark feature

### DISCUSSION THREAD

**Discussion by Tanay27102002** [SE-Group89/WolfJobs#4] [General]
Please check the bookmark feature I have added

**Comment by aayushbrahmbhatt** [comment]
Testing it!

**Comment by aayushbrahmbhatt** [comment]
Might need to do some modifications from the backend @naisargshah please check the routes and the modular for it. I guess there are few mistakes in it

**Comment by naisargshah** [comment]
I am looking into the matter and working on it.
